app/models/performance_dashboards/base.rb

```diff
@@ -164,11 +164,12 @@ def yn(boolean)
   end
 
   # An entry is an enrollment where the entry date is within the report range, and there are no entries in the
-  # specified project types for the prior 24 months.
+  # specified project types for specified inactivity period (default is 24 months).
   def entries
+    inactivity_start_date = @filter.start_date - @filter.inactivity_days.days
```

Comment:
To handle leap years @awisema

```suggestion
    duration = @filter.inactivity_days > 90 ? @filter.inactivity_days.days.in_months :  @filter.inactivity_days.days
    inactivity_start_date = @filter.start_date - duration
```


that way we end up subtracting 24 months instead of (365*2) days